src/selectivity-single.js

```diff
@@ -35,10 +35,11 @@ function SingleSelectivity(options) {
                 deltaUp = Math.min(Math.max(bottom - $(window).height(), 0), top + selectHeight);
             }
 
+            var width = (typeof $selectEl.outerWidth !== 'undefined') ? $selectEl.outerWidth() : $selectEl.width();
```

Comment:
You can use a simpler version:

    var width = ($selectEl.outerWidth ? $selectEl.outerWidth() : $selectEl.width());

(typeof is never necessary if you want to check for undefined, and in this case even just checking for a truthy value is enough)